src/content/app-architecture/case-study/index.md

```diff
@@ -82,48 +82,46 @@ Data layer objects (repositories and services) aren't tied to a single feature,
 while UI layer objects (views and view models) are.
 The following is how the code is organized within the Compass application.
 
-```plaintext
-lib
-├─┬─ ui
-│ ├─┬─ core
-│ │ ├─┬─ ui
-│ │ │ └─── <shared widgets>
-│ │ └─── themes
-│ └─┬─ <FEATURE NAME>
-│   ├─┬─ view_model
-│   │ └─── <view_model class>.dart
-│   └─┬─ widgets
-│     ├── <feature name>_screen.dart
-│     └── <other widgets>
-├─┬─ domain
-│ └─┬─ models
-│   └─── <model name>.dart
-├─┬─ data
-│ ├─┬─ repositories
-│ │ └─── <repository class>.dart
-│ ├─┬─ services
-│ │ └─── <service class>.dart
-│ └─┬─ model
-│   └─── <api model class>.dart
-├─── config
-├─── utils
-├─── routing
-├─── main_staging.dart
-├─── main_development.dart
-└─── main.dart
-
-// The test folder contains unit and widget tests
-test
-├─── data
-├─── domain
-├─── ui
-└─── utils
-
-// The testing folder contains mocks other classes need to execute tests
-testing
-├─── fakes
-└─── models
-```
+<FileTree>
+
+- lib/
+  - ui/
+    - core/
+      - ui/ 
+        - <shared_widgets>
+      - themes/
+    - <feature_name>/
+      - view_model/
+        - <view_model_class>.dart
+      - widgets/
+        - <feature_name>_screen.dart
+        - <other_widgets>
+  - domain/
+    - models/
+      - <model_name>.dart
+  - data/
+    - repositories/
+      - <repository_class>.dart
+    - services/
+      - <service_class>.dart
+    - model/
+      - <api_model_class>.dart
+  - config/
+  - utils/
+  - routing/
+  - main_staging.dart
+  - main_development.dart
+  - main.dart
+- test/ // The test folder contains unit and widget tests
```

Comment:
Since the comments are already styled differently, can we drop the `//` or do you think they're still helpful visually?